packages/connect-examples/expo-example/shim.js

```
@@ -16,7 +16,16 @@ if (typeof process === 'undefined') {
 }
 
 process.browser = false;
-if (typeof Buffer === 'undefined') global.Buffer = require('buffer').Buffer;
+
+// 修复 Buffer 的导入方式，确保与 webpack fallback 兼容
+if (typeof Buffer === 'undefined') {
+  try {
+    const { Buffer: BufferPolyfill } = require('buffer');
+    global.Buffer = BufferPolyfill;
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)